jetty-start/pom.xml

```diff
@@ -44,11 +43,15 @@
             <configuration>
               <createDependencyReducedPom>false</createDependencyReducedPom>
               <createSourcesJar>true</createSourcesJar>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>shaded</shadedClassifierName>
               <filters>
                 <filter>
                   <artifact>org.eclipse.jetty:jetty-util</artifact>
                   <includes>
                     <include>org/eclipse/jetty/util/JavaVersion*</include>
+                    <include>org/eclipse/jetty/util/TopologicalSort*</include>
+                    <include>org/eclipse/jetty/util/component/*</include>
```

Comment:
Adding /component/* is a non-starter.  
This introduces a bunch of other jetty-util features.
Remove ContainerLifeCycle and Dumpable from TopologicalSort